### PR TITLE
Removed `$` in syntaxes

### PR DESCRIPTION
**Pull request by jcohenadad**
When copy/pasting it creates an error (some people could get confused)

### CODE REVIEW

File: computing-resources/neuropoly/data/git-datasets.md [hunk 1]
```diff
@@ -83,7 +83,10 @@ You should also make sure to [configure git annex](./git-annex.md#global-git-con
 To see what datasets you have available, use `info`, for example:
 
 ```
-$ ssh git@data.neuro.polymtl.ca info
+ssh git@data.neuro.polymtl.ca info
+```
+And the output would look like this:
+```
```
Comment (kousu) [inline computing-resources/neuropoly/data/git-datasets.md:89]: Pretty sure this needs a linebreak to parse ?

````suggestion
ssh git@data.neuro.polymtl.ca info
```

And the output would look like this:
```
````

**Review by kousu — APPROVED** [review]
So just for some context, here I was following the convention that `$ ` -- the command prompt itself -- means the entire quote is what you would see on your screen, input *and* output, whereas not putting a `$` meant it was a script you could run directly.

Most of these blocks could have been script-like, and you made them so; but someone might not want to run them directly either, because a lot of these chains could fail in lots of ways. Including the `$` should force users to read and understand what they are typing before doing it blindly...

File: computing-resources/neuropoly/data/git-datasets.md [hunk 2]
```diff
@@ -123,17 +126,21 @@ Despite not being hosted on Github, we are still using a [pull-request workflow]
...
 
 The *first* time before uploading, verify you have access with `info`. You need "W" (for "Write") permission, like this:
 
 ```
-$ ssh git@localhost info datasets/uk-biobank
+ssh git@localhost info datasets/uk-biobank
+```
+
+The output would look like:
+```
```
Comment (kousu) [inline computing-resources/neuropoly/data/git-datasets.md:143]: https://github.com/neuropoly/intranet.neuro.polymtl.ca/pull/31#pullrequestreview-821055047

File: computing-resources/neuropoly/data/git-datasets.md [hunk 3]
```diff
@@ -347,7 +354,11 @@ cat id_rsa.zamboni.pub | ssh git@data.neuro.polymtl.ca keys add zamboni
 You can also paste the key in, followed by `ctrl-d`; this looks like:
 
 ```
-$ ssh git@data.neuro.polymtl.ca keys add zamboni
+ssh git@data.neuro.polymtl.ca keys add zamboni
+```
+
+The output looks like:
+```
```
Comment (kousu) [inline computing-resources/neuropoly/data/git-datasets.md:361]: This was also on purpose, re: https://github.com/neuropoly/intranet.neuro.polymtl.ca/pull/31#pullrequestreview-821055047

**Comment by jcohenadad** [comment]
> So just for some context, here I was following the convention that `$ ` -- the command prompt itself -- means the entire quote is what you would see on your screen, input _and_ output, whereas not putting a `$` meant it was a script you could run directly.
> 
> Most of these blocks could have been script-like, and you made them so and that's probably an improvement; but maybe we should think that someone might not want to run them directly either: a lot of these chains could fail in lots of ways, and this isn't meant to be copy-paste-success, it's meant to be a guideline for someone to read along with and understand. Including the `$` has the useful side-effect of forcing users to read and understand what they are typing before doing it blindly... I mean, do we really want to encourage people to copy-paste `git annex get sub-karo*` when most datasets don't have such subjects, or ditto `git add path/to/new/file`?

thank you for the context, @kousu 

yes, i figured that was the intention behind using the '$' convention, but at the end of the day there will be a lot more copy/pasting via mouse double-click (ie: refresher of something already understood) than basic interpretation of the commands.

**Pull request merged**